RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforDevice.h

```diff
@@ -234,17 +234,30 @@ namespace pixelCPEforDevice {
     if (cp.minCol[ic] == 0 || cp.maxCol[ic] == uint32_t(detParams.nCols - 1))
       cp.ysize[ic] = -cp.ysize[ic];
 
+    /* original code, kept for documentation
+
     // apply the lorentz offset correction
     float xoff = 0.5f * float(detParams.nRows) * comParams.thePitchX;
```

Comment:
was the original comment actually correct? Isn't only the old L245 with `shiftX` the LA offset, while the rest is a translation wrt center?

Reply:
I have no idea, but if can suggest a better description I'm happy to fix it

Reply:
I'm not sure of the full provenance/authorship of the comments.

@ferencek please check to see if the comments are descriptive. Thanks.

Reply:
the first commit of the original CUDA code (circa 2018) had
```c++
  // apply the lorentz offset correction
  auto xPos = detParams.shiftX + comParams.thePitchX*(0.5f*float(mx)+float(phase1PixelTopology::xOffset));
  auto yPos = detParams.shiftY + comParams.thePitchY*(0.5f*float(my)+float(phase1PixelTopology::yOffset));
```

The other corrections and comments were added over time.